Readme.md

```diff
@@ -36,50 +36,28 @@ git clone https://github.com/aerogear/mobile-cli
 
 ```
 glide install
-go build cmd/mobile/main.go
+make build
 ```
 To test run
 
 ```
-./main
+./mobile
```

Comment:
@pb82 in https://github.com/aerogear/mobile-cli/pull/10 I proposed name to be `om`. 
It's short and there are no conflicts in the unix package tree.

@maleck13 WDYT?

Reply:
not convinced as this will be targeting the k8s plugin. I think
``` kubectl plugin mobile ``` or hopefully ``` kubectl mobile``` at some point is nice than ```kubectl om ```